platform/android/java/lib/src/org/godotengine/godot/GodotIO.java

```diff
@@ -306,6 +309,19 @@ public int getScreenOrientation() {
 		}
 	}
 
+	public int getDisplayRotation() {
```

Comment:
Can you validate the issue raised in https://github.com/godotengine/godot/pull/98709 for the use of `windowManager.getDefaultDisplay().getRotation()` doesn't apply here as well.

Reply:
Thanks for the review!

While Display.getRotation() can be unreliable for determining the *Vulkan rendering pre-transform*, it still reflects the *logical orientation of the display surface*.

Orienting camera images requires aligning the camera sensor with this *logical display orientation*. The standard calculation uses sensor orientation and the *display's rotation*, not the Vulkan swapchain transform (which is for *rendered output* transformation).

So, using Display.getRotation() (or equivalent logical display info) for camera orientation is the correct approach, separate from Vulkan rendering details.